qna/DE-TSE-support.md

```diff
@@ -2,10 +2,18 @@
 
 Welche TSE-Anbieter werden von fiskaltrust unterstützt?
 
+## Question
+
+Gibt es eine eigene TSE von fiskaltrust?
+
 ## Metadata tags
 
 lang-de, market-de, middleware, PosCreator, PosDealer, Consultant
 
 ## Answer
 
+fiskaltrust ist ein Middleware-Anbieter und bietet keine eigene TSE an, um möglichst alle am Markt verfügbaren TSE mit der fiskaltrust.Middleware unterstützen zu können.
+
+Das ft.Interface bietet daher eine technische und architektonische Unabhängigkeit von der TSE. Über das ft.Portal haben die Kunden die Möglichkeit, ihre gewählte TSE zu konfigurieren und diese auch jederzeit zu wechseln, sowohl von einer Offline-TSE zu einer Online-TSE, also auch umgekehrt.
+
 Eine Übersicht über unterstützte TSE finden sie hier: https://github.com/fiskaltrust/productdescription-de-doc/blob/master/compliance-as-a-service/produkte/4445-0003-lokal-installierte-middleware.md
```

Comment:
change link to: https://github.com/fiskaltrust/productdescription-de-doc/blob/master/compliance-as-a-service/produkte/4445-0003-lokal-installierte-middleware.md#interoperabilit%C3%A4t

Reply:
thank you, updated the link